x-pack/legacy/plugins/ml/public/datavisualizer/file_based/components/file_datavisualizer_view/index.js

```diff
@@ -5,4 +5,5 @@
  */
 
 
-export { FileDataVisualizerView, MODE } from './file_datavisualizer_view';
+export { FileDataVisualizerView } from './file_datavisualizer_view';
+export { MODE } from './constants';
```

Comment:
looks like this export isn't needed as mode has been moved out of `file_datavisualizer_view` and is only used by `BottomBar` which is now importing it directly from `constants`

Reply:
Fixed in https://github.com/elastic/kibana/pull/50147/commits/3df5ebf3cfa08820f23a11cbb3178712a8d1c71d